.gitattributes

```diff
@@ -0,0 +1 @@
+wait-for text eol=lf
```

Comment:
Do we need this? I haven't had issues so far with line-endings nor heard anyone else about it.

If we need it, can we then set it for the entire repo? It would be the preferred line-ending for all files anyway, nothing specific to the wait-for script.

```suggestion
*	text=auto

```
https://www.git-scm.com/docs/gitattributes#_end_of_line_conversion

Reply:
I'm sometimes working on Windows and I didn't configure my eol conversion globally. The local working copy failed to work in combination with the Docker container based tests. I'm going to move this change to a dedicated issue and we can continue the discussion over there.